recipes/magnon/recipe.yaml

```diff
@@ -0,0 +1,60 @@
+context:
+  name: magnon
+  version: "1.0.0"
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/8e/81/eb86a9e28708859cdbbc7dcf6f4bc40ce83343cd6f8b3a8177b5678f052c/magnon-1.0.0.tar.gz
+  sha256: 2f749c4fdd68c7c36162e0f0abcb6e1765f900ef680b746c57324878d7aa85a8
+
+build:
+  script: python -m pip install . -vv
+  number: 0
+  skip:
+     - match(python, "<3.12")
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools
+    - numpy
+    - matplotlib
+    - ase
+    - spglib
+    - pytest
+  run:
+    - python
+    - numpy
+    - matplotlib
+    - ase
+    - spglib
+    - pytest
+
+tests:
+  - python:
+      imports:
+      - magnon
+      pip_check: true
+
+about:
+  homepage: https://github.com/MPA2suite/Magnon
+  summary: 'Magnon bandstructures in Python'
+  description: |
+    Magnon is a Python library integrated with ASE for fast and accurate simulations of linear spin-wave theory band structures. 
+    The code is versatile and applicable to a broad range of magnetic systems, including: collinear and non-collinear materials, 
+    systems with Q-commensurate magnetic ordering, and cases where the exchange coupling includes off-diagonal Cartesian components 
+    such as the Dzyaloshinskii-Moriya interaction (DMI) and Kitaev interactions. The library includes a set of propedeutically 
+    structured tutorials, and a self-explanatory API that can be straightforwardly integrated into computational workflows to 
+    simulate magnon properties with quantitative accuracy.
+  license: 'LicenseRef-ASL'
```

Comment:
B/c we are not lawyers and our volunteer time to read and understand custom licenses is limited I would like to ask if you can convince upstream to use an OSI approved license. Or, if you cannot, at least write here that this license is 100% compatible with ab OSI approved license.